docs/src/refs.bib

```diff
@@ -0,0 +1,48 @@
+@article{neal2011mcmc,
+  title={Mcmc using hamiltonian dynamics (handbook of markov chain monte carlo) ed s brooks et al},
```

Comment:
```suggestion
  title={MCMC using Hamiltonian dynamics (Handbook of Markov Chain Monte Carlo) ed s Brooks et al},
```
Also the `ed s Brooks et al` should probably be formatted differently somehow, not sure what the best way is.